galley/pkg/config/analysis/analyzers/testdata/virtualservice_destinationhosts.yaml

```diff
@@ -60,3 +69,27 @@ spec:
     - destination:
         host: external-reviews.org  # Referring to a ServiceEntry host is valid and should not generate an error
                                     # Since this is an "external" service, subset is omitted
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews-bookinfo-eu
+  namespace: default
+spec:
+  http:
```

Comment:
this is not a valid rule. missing hosts

Reply:
Correct. For unit tests, the resources defined here don't need to be complete: we're not running all of the analysis & validation, just one analyzer at a time.